File: 1121.c
#include <stdio.h>
#include <math.h>
#include <stdlib.h>
  
int main(){
   
    int x1, y1, x2, y2;
    scanf("%d %d %d %d", &x1, &y1, &x2, &y2);
      
    if(x1==x2 || y1==y2 || abs(x2-x1) == abs(y2-y1)){
        printf("YES");
    }else printf("NO");
      
    return 0;
}

File: 1121.py
k1, k2 = map(int, input().split())
l1, l2 = map(int, input().split())
 
if k1 == l1 or k2 == l2:
    print("YES")
elif abs(l1-k1) == abs(l2-k2):
    print("YES")
else:
    print("NO")
